src/client/rewards/Details/Details.js

```diff
@@ -0,0 +1,182 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Button, Modal, Checkbox } from 'antd';
+import { injectIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { isEmpty } from 'lodash';
+import detailsData from './detailsData';
```

Comment:
import detailsData from './detailsData';
change to:
import getDetailsMessages from './detailsMsgData';

Reply:
done